### PR TITLE
feat(k8s-manager): add workflow name to nodes names

### PR DESCRIPTION
**Pull request by JYisus**
## Description
Add workflow name in the deployment name of the version nodes.

## Motivation and Context
Two workflows of the same version can have 2 nodes with the same name, so adding the workflow in the deployment name makes identifying which node is easier when debugging. Until now, the node id was used for this purpose, but the workflow names could be more helpful.

### CODE REVIEW

**Comment by dfinteligenz** [comment]
Do we have validation for checking that there are not repeated names on the nodes of the same workflow?

If si, this looks good to me :D

**Review by dfinteligenz — DISMISSED** [review]

**Comment by kafkaphoenix** [comment]
looks good to me too! i prefer strings to ids

**Review by alexrodfe — APPROVED** [review]

File: engine/admin-api/domain/usecase/krt/validator/validator_v2.go [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+	"errors"
 	"fmt"
 
 	"github.com/konstellation-io/kre/engine/admin-api/domain/usecase/krt"
 	"github.com/konstellation-io/kre/engine/admin-api/domain/usecase/logging"
 )
 
+var ErrRepeatedNodeName = errors.New("node names should be unique per workflows")
+
 type ValidatorV2 struct {
```
Comment (dfinteligenz) [inline engine/admin-api/domain/usecase/krt/validator/validator_v2.go:13]: We should remove reference to the V1 and V2 versions, as the V1 is already deprecated, and we are working with the V2 only.

File: engine/admin-api/domain/usecase/krt/validator/validator_v2.go [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/konstellation-io/kre/engine/admin-api/domain/usecase/krt"
 	"github.com/konstellation-io/kre/engine/admin-api/domain/usecase/logging"
 )
 
+var ErrRepeatedNodeName = errors.New("node names should be unique per workflows")
```
Comment (kafkaphoenix) [inline engine/admin-api/domain/usecase/krt/validator/validator_v2.go:11]: i would move the error to an error file

**Review by kafkaphoenix — APPROVED** [review]

**Review by kafkaphoenix — APPROVED** [review]

**Review by dfinteligenz — APPROVED** [review]
LGTM

**Pull request merged**